cobra/core/Model.py

```diff
@@ -118,6 +117,45 @@ def description(self, value):
         self.name = value
         warn("description deprecated", DeprecationWarning)
 
+    @property
+    def medium(self):
+        return {rxn.id: rxn.bounds for rxn in
```

Comment:
I don't really see a way around this, short of requiring that users manually specify the reactions they're interested in.

One option could be to only look for boundary reactions with reaction.bounds that allow metabolites to be created? but then we loose the ability to set, for instance, the max. CO2 production rate. 

My approach was to return the bounds for all of the reactions on the boundary, which will necessarily include export reactions, and then let the user handle them appropriately.